### ISSUE TITLE
Fall without any reason under Pyramid of Settra design, no RF used while 

### ISSUE THREAD

**Issue by RogertG** [WarEmu/WarBugs#20598]
**Expected behavior and actual behavior:**

Expected behavior: Not to fall under the floor
Actual behavior: In certain  (could not identify the cause) situation (happened only once) falling under the model's design.

**Steps to reproduce the problem:**

Please see the attached video. 

Data for MySql identification:

Pyramid of Settra
Date: 2022-12-02
Time: 11:31:12

https://killboard.returnofreckoning.com/scenario/727355e1-6d50-43ca-9f03-b0257de7dfe7
User: Dhardor
Class: WP
Faction Order

**Testing Screenshots/Videos/Evidences (always needed):**

https://youtu.be/xlyqt4R8nzQ

**Comment by lilimx** [comment]
Hey,
Where excatly did you fall under the map?

**Comment by RogertG** [comment]
On a perpendicular axix (eg. plan X-Y) from exactly where I am standing in the video and rotating my camera.

**Comment by lilimx** [comment]
Hey,

I mean....you were inside the pyramide...and then you've glitched through, where have you been in that moment.

**Comment by lilimx** [comment]
Hey,

I am closing this ticket now as we have not received any further information regarding the problem.
If you are the opinion that the issue still persists, please open a new ticket with a clearer description .
